apps/src/code-studio/pd/foorm/editor/form/FormEditorManager.jsx

```diff
@@ -236,7 +238,9 @@ class FormEditorManager extends React.Component {
             previewQuestions={this.state.previewQuestions}
             previewErrors={this.listPreviewErrors()}
             forceRerenderKey={this.state.forceRerenderKey}
-            renderHeaderTitle={() => this.renderHeaderTitle()}
+            headerTitle={this.renderHeaderTitle()}
```

Comment:
is the title format going to be different in the library editor other than using "Library..." instead of "Form..."?

if the rendering format will be the same (`<div><h2><h3>`) it would be better to pass in any necessary text so you only need the rendering code in one place.

Reply:
It's going to be different, we'll include a library name and version as well as a library question name in the library editor.